picosdk/ps5000a.py

```diff
@@ -180,12 +314,60 @@ class PS5000A_TRIGGER_CHANNEL_PROPERTIES_V2(Structure):
                 ("thresholdLower", c_int16),
                 ("thresholdLowerHysteresis", c_uint16),
                 ("channel", c_int32)]
+
+class PS5000A_TRIGGER_INFO(Structure):
+    _pack_ = 1
+    _fields_ = [("status", PICO_STATUS),
+                ("segmentIndex", c_uint32),
+                ("triggerIndex", c_uint32),
+                ("triggerTime", c_int64),
+                ("timeUnits", c_int16)],
+                ("reserved0", c_int16)],
+                ("timeStampCounter", c_uint64)]
 				
 class PS5000A_CONDITION (Structure):
 	_pack_ = 1
 	_fields_ = [("source", c_int32),
 				("condition", c_int16)]
+
+class PS5000A_PWQ_CONDITIONS (Structure):
+	_pack_ = 1
+	_fields_ = [("channelA", PS5000A_TRIGGER_STATE),
+                ("channelB", PS5000A_TRIGGER_STATE),
+                ("channelC", PS5000A_TRIGGER_STATE),
+                ("channelD", PS5000A_TRIGGER_STATE),
+                ("external", PS5000A_TRIGGER_STATE),
+				("aux", PS5000A_TRIGGER_STATE)]
 				
+class PS5000A_TRIGGER_CONDITIONS (Structure):
+	_pack_ = 1
+	_fields_ = [("channelA", PS5000A_TRIGGER_STATE),
+                ("channelB", PS5000A_TRIGGER_STATE),
+                ("channelC", PS5000A_TRIGGER_STATE),
+                ("channelD", PS5000A_TRIGGER_STATE),
+                ("external", PS5000A_TRIGGER_STATE),
+				("aux", PS5000A_TRIGGER_STATE),
+                ("pulseWidthQualifier", PS5000A_TRIGGER_STATE)]
+
+class PS5000A_DIRECTION (Structure):
+	_pack_ = 1
+	_fields_ = [("source", PS5000A_CHANNEL),
+                ("condition", PS5000A_TRIGGER_STATE)]
+
+class PS5000A_CONDITION (Structure):
```

Comment:
PS5000A_CONDITION is defined twice (see line 328). This definition is the wrong one since this structure has no field 'mode' according to the manual. In order to make triggering work, this class must be deleted.